contracts/ATM/ATMGovernance.sol

```diff
@@ -0,0 +1,163 @@
+pragma solidity 0.5.17;
+
+// External Libraries
+import "@openzeppelin/contracts/lifecycle/Pausable.sol";
+import "@openzeppelin/contracts/utils/Address.sol";
+
+// Common
+import "../util/AddressArrayLib.sol";
+import "../util/AddressLib.sol";
+
+// Contracts
+import "@openzeppelin/contracts/access/roles/SignerRole.sol";
+
+// Interfaces
+import "./IATMGovernance.sol";
+
+/**
+    @notice This contract is used to modify Risk Settings, CRA or DataProviders for a specific ATM.
+    @author develop@teller.finance
+ */
+contract ATMGovernance is Pausable, SignerRole, IATMGovernance {    
```

Comment:
How do we deal with the pause feature? Who will able to call the pause function?
Do we need to inject the ATMSettings (it has the pause function) to verify whether the ATM is paused or not?

Reply:
Yes, based on yesterday's conversation I removed the Pausable capability and I'll inject ATMSettings

Reply:
ATMGovernance is currently a contract where we store a specific ATM settings. I was following the rule to leave configurations/settings non-pausable in case we need to modify a value to fix an issue, so I removed Pausable capability and wasn't planning to add Settings. We might need a discussion around this.